defaults/main.yml

```diff
@@ -257,6 +257,7 @@ dataverse:
     argument: '-DcompilerArgument=-Xlint:unchecked test -P all-unit-tests'
   usermgmtkey: burrito
   version: '5.13'
+#  robots_txt_url: https://guides.dataverse.org/en/latest/_downloads/3a5cd7a283eecd5e93289e30af713554/robots.txt
```

Comment:
I'm not sure if this is our usual practice... to comment out variables like this. But I guess it's fine.

Reply:
Where would you put the address of the recommended robots.txt ?

Reply:
okay, then I will merge this...